src/main/java/com/softserve/academy/spaced/repetition/logger/LogExceptionAspect.java

```diff
@@ -3,25 +3,24 @@
 import org.aspectj.lang.JoinPoint;
 import org.aspectj.lang.annotation.AfterThrowing;
 import org.aspectj.lang.annotation.Aspect;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
+import org.slf4j.Logger;
 
 @Aspect
 @Component
-public class LogException {
-    @Autowired
-    Logger logger;
+public class LogExceptionAspect {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogExceptionAspect.class);
 
     @AfterThrowing(value = "execution(* com.softserve.academy.spaced.repetition.controller.*.*(..))", throwing = "ex")
     public void loggingExceptionsFromControllers(JoinPoint joinPoint, Exception ex) {
-        System.out.println(logger);
-        logger.log("Exception in controller = " + joinPoint.getThis() + ", method = " + joinPoint.getSignature());
+        LOGGER.error("Exception in controller = {} method = {}", joinPoint.getThis(), joinPoint.getSignature(), ex);
     }
 
     @AfterThrowing(value = "execution(* com.softserve.academy.spaced.repetition.service.*.*(..))", throwing = "ex")
     public void loggingExceptionsFromServices(JoinPoint joinPoint, Exception ex) {
-        System.out.println(logger);
-        logger.log("Exception in service = " + joinPoint.getThis() + ", method = " + joinPoint.getSignature());
+        LOGGER.error("Exception in controller = {} method = {}", joinPoint.getThis(), joinPoint.getSignature(), ex);
```

Comment:
This is why we use code review. Please be careful - you changed one word. Please revert to "...service".

Reply:
Sorry i'm just in a hurry.